test/integration/suites/upgrade/versions.txt

```diff
@@ -3,3 +3,4 @@
 1.4.2
 1.4.3
 1.4.4
+1.4.5
```

Comment:
1.5.0 is missing here.

Reply:
Ah, you're right. I always get confused which versions belong in here.